docs/rules/no-this-in-before-route-enter.md

```diff
@@ -0,0 +1,72 @@
+# vue/no-this-in-before-route-enter
+
+> This rule prevents usage this in the "beforeRouteEnter" because this is undefined there. https://router.vuejs.org/guide/advanced/navigation-guards.html#in-component-guards
+
+## Rule Details
+
```

Comment:
Could you write a description of the rules here? Documents should automatically update their headers and footers with the `npm run update` command.